alembic/versions/104d4a614d95_add_performancestat.py

```diff
@@ -5,32 +5,33 @@
 Create Date: 2023-05-27 02:59:14.032373
 
 """
-from alembic import op
 import sqlalchemy as sa
 
+from alembic import op
 
 # revision identifiers, used by Alembic.
-revision = '104d4a614d95'
-down_revision = 'b2dc41850120'
+revision = "104d4a614d95"
+down_revision = "b2dc41850120"
```

Comment:
Black should be configured to ignore this directory

Reply:
Addressed in: https://github.com/OpenAdaptAI/OpenAdapt/compare/71f37a4e2cc772cacec72c40ea14d7d15a54740e..1380d35e42f6651dc6f13fbc1dc652eee221f762